ppr-ui/src/components/common/HomeLocationInfo.vue

```diff
@@ -0,0 +1,231 @@
+            persistent-hint
+          />
+        </v-col>
+      </v-row>
+
+      <v-row no-gutters>
+        <v-col cols="4" class="pr-2">
+          <v-text-field
+            filled
+            id="township"
+            label="Township"
+            v-model="locationInfo.township"
+            :rules="maxLength(2)"
+            persistent-hint
+          />
+        </v-col>
+
+        <v-col cols="4" class="px-1">
+          <v-text-field
+            filled
+            id="Range"
+            label="Range"
+            v-model="locationInfo.range"
+            :rules="maxLength(2)"
+            persistent-hint
+          />
+        </v-col>
+
+        <v-col cols="4" class="pl-2">
+          <v-text-field
+            filled
+            id="meridian"
+            label="Meridian"
+            v-model="locationInfo.meridian"
+            :rules="maxLength(3)"
+            persistent-hint
+          />
+        </v-col>
+      </v-row>
+
+      <v-row no-gutters>
+        <v-col cols="6" class="pr-2">
+          <v-text-field
+            filled
+            id="parcel"
+            label="Parcel"
+            v-model="locationInfo.parcel"
+            :rules="locationInputRules(10, 'Enter a parcel block')"
+            persistent-hint
+          />
+        </v-col>
+
+        <v-col cols="6" class="pl-2">
+          <v-text-field
+            filled
+            id="block"
+            label="Block"
+            v-model="locationInfo.block"
+            :rules="locationInputRules(10, 'Enter a block')"
+            persistent-hint
+          />
+        </v-col>
+      </v-row>
+
+      <v-row no-gutters>
+        <v-col cols="12">
+          <v-textarea
+            filled
+            id="exceptPlan"
+            label="Except Plan"
+            height="100"
+            v-model="locationInfo.exceptPlan"
+            :rules="maxLength(80)"
+            persistent-hint
+          />
+        </v-col>
+      </v-row>
+    </v-form>
+  </v-card>
+</template>
+
+<script lang="ts">
+/* eslint-disable no-unused-vars */
+import { computed, defineComponent, reactive, toRefs, watch } from '@vue/composition-api'
+import { MhrLocationInfoIF } from '@/interfaces'
+import { useInputRules } from '@/composables/useInputRules'
+/* eslint-disable no-unused-vars */
+
+export default defineComponent({
+  name: 'HomeLocationInfo',
+  emits: ['updateLocationInfo', 'updateLocationValid'],
+  props: {
+    validate: { type: Boolean, default: false }
+  },
+  setup (props, context) {
+    const {
+      customRules,
+      maxLength,
+      required
+    } = useInputRules()
+
+    const localState = reactive({
+      homeLocationInfoValid: false,
+      locationInfo: {
+        lot: '',
+        landDistrict: '',
+        plan: '',
+        districtLot: '',
+        partOf: '',
+        section: '',
+        township: '',
+        range: '',
+        meridian: '',
+        parcel: '',
+        block: '',
+        exceptPlan: ''
+      },
```

Comment:
Let's add a type here `MhrLocationInfoIF`.

Reply:
For sure!

Reply:
updated